src/main/resources/swagger/api-docs.yaml

```diff
@@ -98,6 +98,34 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/admin/v2/user/{userId}/repairCloudAccess:
+    get:
+      tags:
+        - Admin
+      summary: Ensures that a user's proxy group exists and that it is added to any google groups that it should be in.
+      operationId: adminRepairUserCloudAccess
+      parameters:
+        - name: userId
+          in: path
+          description: User ID of the user to have their cloud access repaired
+          required: true
+          schema:
+            type: string
+      responses:
+        204:
+          description: User access was repaired successfully (as long as the group sync messages succeed)
+        403:
+          description: You do not have admin privileges
+          content: { }
```

Comment:
Should the content here (and with the 404) be the same as the 500 error?
```
           application/json:
              schema:
                $ref: '#/components/schemas/ErrorReport'
```

Reply:
I think this is unnecessary for 403/404, the reason the error report is there for 500 is because that is an unexpected error where it could be helpful to include additional information. For 404/403 it is obvious what the problem is.